raiwidgets/raiwidgets/fairness_dashboard.py

```diff
@@ -93,24 +106,53 @@ def __init__(
         self.fairness_metrics_module = metrics_module
 
         def metrics():
+            """ 
+            Note:
+                This function always calculates the error_function, if available,
+                so that the value is cached in the MetricsCache
+
+                Params
```

Comment:
can you use the same syntax as elsewhere with `:param:` etc.?

Reply:
I didn't touch some of the params, but there was no documentation before. I don't think I can write the docs properly for them. Can you write them for binVector and modelIndex?

Reply:
I see now that these aren't actually args as such, but rather available through the request. You can write a description for them, though. 
binVector is the sensitive features binning vector